src/main/java/pw/kaboom/extras/commands/CommandRainbow.java

```diff
@@ -0,0 +1,31 @@
+package pw.kaboom.extras.commands;
+
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import net.kyori.adventure.text.minimessage.MiniMessage;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import javax.annotation.Nonnull;
+
+public final class CommandRainbow implements CommandExecutor {
+    private static MiniMessage mm = MiniMessage.miniMessage();
```

Comment:
It might be best to create a class with a public static final MiniMessage instance so we prevent unnecessary duplicated calls to MiniMessage.miniMessage()